scanpipe/tests/pipes/test_d2d.py

```diff
@@ -652,3 +653,127 @@ def test_scanpipe_pipes_d2d_map_javascript_path(self):
         relation = self.project1.codebaserelations.all()
         self.assertEqual(from_resource, relation[0].from_resource)
         self.assertEqual(from_resource, relation[1].from_resource)
+
+    def test_scanpipe_pipes_d2d_map_javascript_colocation(self):
+        to_dir1 = (
+            self.project1.codebase_path / "to/project.tar.zst-extract/osgi/marketplace/"
+            "intelligent robotics platform.lpkg-extract/"
+            "com.example.adaptive.media.web-0.0.5.jar-extract/META-INF/"
+            "resources/adaptive_media/js"
```

Comment:
Is there a value keeping that path complexity for the purpose of this unit test?

Reply:
IMO since colocation mapping is based on the relative path of files it would be great to have this tested on the complex path.

Reply:
Make sense.